src/Stan/Inspection/Partial.hs

```diff
@@ -0,0 +1,58 @@
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+Contains all 'Inspection's for partial functions.
+-}
+
+module Stan.Inspection.Partial
+    ( -- * Partial 'Inspection's
+      stan0001
+    , stan0001Inspection
+
+      -- * List of all partial 'Inspection's
+    , partialInspections
+    , partialInspectionsIds
+    ) where
+
+import Stan.Category (partial)
+import Stan.Core.Id (Id (..))
+import Stan.Inspection (Inspection (..), Severity (..))
+
+
+-- | All partial 'Inspection's.
+partialInspections :: [Inspection]
+partialInspections =
+    [ stan0001Inspection
+    ]
+
+-- | All partial 'Inspection's 'Id's.
+partialInspectionsIds :: [Id Inspection]
+partialInspectionsIds =
+    [ stan0001
+    ]
```

Comment:
What do you think about extracting `Id`s from the list of all inspections? We already specify the list of all inspections, so we can get all ids from this list. Or is it better to specify the list of ids explicitly as well?
```suggestion
partialInspectionsIds = map inspectionId partialInspections
```